project_milestone/models/project_task.py

```diff
@@ -0,0 +1,22 @@
+from odoo import models, fields, api
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    milestone_id = fields.Many2one('project.milestone',
+                                   string="Milestones",
+                                   group_expand='_read_group_milestone_ids',
+                                   domain="[('project_id', '=', project_id)]")
+    use_milestones = fields.Boolean(string="Use Milestones",
+                                    related='project_id.use_milestones',
+                                    help="If enabled, then Milestones will be \

```

Comment:
Delete \^M characters

Reply:
It's strange, I don't see any character after '\' no matter what editor I try.

Reply:
I think you must add the " at the end of each line.
```suggestion
                                    help="If enabled, then Milestones will be "
```

Reply:
I think that worked, thanks for the suggestion. I updated it in the latest commit.